numpy/__init__.pyi

```diff
@@ -1,8 +1,91 @@
 # very simple, just enough to start running tests
+#
+import builtins
+from typing import Any, Mapping, List, Optional, Tuple, Union
 
-class ndarray: pass
+from numpy.core._internal import _ctypes
+
+class dtype:
+    alignment: int
+    byteorder: str
+    char: str
+    descr: List[Tuple[str, str]]
+    fields: Optional[Mapping[str, Union[Tuple[dtype, int], Tuple[dtype, int, str]]]]
```

Comment:
It looks like the final `title` field can have any type, so it should probably be keyed as `Any` instead of `str`: https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.dtype.fields.html#numpy.dtype.fields